PWGHF/D2H/TableProducer/candidateCreatorCharmResoReduced.cxx

```diff
@@ -312,7 +312,7 @@ struct HfCandidateCreatorCharmResoReduced {
 
         if constexpr (channel == DecayChannel::DstarTrack) {
           invMassReso = RecoDecay::m(std::array{pVecD, pVecV0Tr}, std::array{massDstar, massProton});
-          registry.fill(HIST("hMassDstarTrack"), invMassReso, ptReso);
+          registry.fill(HIST("hMassDstarTrack"), invMassReso - invMassD, ptReso);
```

Comment:
if you compute the delta Mass, the invariant mass of the resonance `invMassReso` should be computed from the daughter tracks and it should not be the PDG one (otherwise the purpose of calculating the delta mass is not fulfilled, since we do not cancel any momentum resolution)